build/flutter_app.gni

```diff
@@ -374,3 +348,59 @@ template("flutter_aot_app") {
     ]
   }
 }
+
+# Defines a Flutter application
+#
+# Parameters
+#
+#   main_dart (required)
+#     Name of the Dart file containing the main function.
+#
+#   package_name (optional)
+#     Name of the Dart package.
+#
+#   output_name (optional)
+#     Name of output to generate. Defaults to $target_name.flx.
+#
+#   deps (optional)
+#     List of Dart packages the application depends on.
+#
+#   manifest (optional)
+#     Path to the manifest file
+#
+#   analysis_options (optional)
+#     By default, a script to run the analyzer on the contents of the package is
+#     generated in the output directory. This parameter contains the path to an
+#     analysis options file for this target.
+#
+#   disable_analysis (optional)
+#     Prevents the analysis script from being generated.
+template("flutter_app") {
+  if (true) {
```

Comment:
I'm not sure I like the idea of this being toggled by a global flag, but I think it's good to be explicit that there's a choice here.